.gitignore

```diff
@@ -108,7 +108,7 @@ dist
 **/docs
 
 # Lib Folder
-/lib
+# /lib
```

Comment:
What's the actual reason we want to stop using npm deployments and use the git files directly?